platform/android/display_server_android.cpp

```diff
@@ -95,6 +95,17 @@ String DisplayServerAndroid::clipboard_get() const {
 	}
 }
 
+bool DisplayServerAndroid::clipboard_has() const {
+	GodotJavaWrapper *godot_java = OS_Android::get_singleton()->get_godot_java();
+	ERR_FAIL_COND_V(!godot_java, false);
+
+	if (godot_java->has_has_clipboard()) {
+		return godot_java->has_clipboard();
+	} else {
+		return DisplayServer::clipboard_has();
+	}
```

Comment:
I know this pattern is already in the code, but it seems incorrect to me as the default method implementation prints out a `not supported` warning, which is not the case in this scenario.

No actions needed on your part, I may just revisit this logic later and clean it up.